docs/release-v1-perf-scale-benchmarks.md

```diff
@@ -0,0 +1,71 @@
+### Benchmarks
+
+This document shares some of the performance benchmarks observed as part of the v1 release.
+It will talk about what this means for user's perf and scale story.
```

Comment:
```diff
-It will talk about what this means for user's perf and scale story.
+It will talk about what this means for an end-user's perf and scale story.
```